Lib/distutils/unixccompiler.py

```diff
@@ -227,7 +227,7 @@ def runtime_library_dir_option(self, dir):
         if sys.platform[:6] == "darwin":
             # MacOSX's linker doesn't understand the -R flag at all
             return "-L" + dir
-        elif sys.platform[:7] == "freebsd":
+        elif sys.platform.startswith("freebsd"):
```

Comment:
I do not see why FreeBSD has a special treatment it should work the same as linux here

Reply:
I'm sorry, I don't know this code and I never understood what is rpath. If you consider that the code is outdated/can be simplified, would you mind to open a new issue? Maybe even write a PR to implement your proposed change?

Reply:
It is simple the only thing needed it dropping the block and let it fall into the else like linux

Reply:
I created https://github.com/python/cpython/pull/5233 : would you mind to review it *and test it*?

Reply:
I reverted this change and wrote PR #5233 instead.